### DISCUSSION TITLE
How to Install Plugin to Openutau

### DISCUSSION THREAD

**Discussion by UncleGarbage** [stakira/OpenUtau#1091] [Q&A]
I can't find a place that said "plugin" so i don't know how to put plugin

**Comment by nagotown** [comment]
put the plugin in the plugins folder and you should be all set 👍 

![explorer_06L2ylBJSB](https://github.com/stakira/OpenUtau/assets/35883871/c672441c-c900-4d5f-ba2c-7da26833e436)
